frontend/jupyter/src/app/types/volume/interfaces.ts

```diff
@@ -35,6 +39,8 @@ export interface Volume {
 export interface PVCProcessedObject extends PVCResponseObject {
   deleteAction?: string;
   editAction?: string;
+  closePVCViewerAction?: string;
+  openPVCViewerAction?: string;
```

Comment:
Once again upstream, but why need two? Wouldnt the action be either open or closed? Unless they can be both at once?

Reply:
string might not be a boolean. I haven't seen the pvc viewer in action since we dont have that in our cluster yet.
There might be multiple states for these actions.